cypress/e2e/hub/collections.cy.ts

```diff
@@ -48,6 +48,44 @@ describe('Collections- List View', () => {
     });
   });
 
+  it('should approve and sign collection version', () => {
+    const namespace = `sign_namespace_${randomString(3, undefined, { isLowercase: true })}`;
+    cy.createNamespace(namespace);
+    const collection = randomString(5, undefined, { isLowercase: true }).replace(/\d/g, '');
+    cy.galaxykit('namespace create', namespace);
+    cy.galaxykit('collection upload', namespace, collection, '2.0.0');
+    cy.navigateTo('hub', Approvals.url);
+    cy.get('[aria-label="Type to filter"]').type(collection);
+    cy.wait(5000);
```

Comment:
If we could replace this one with a `cy.get(` that would be ideal. Since a hardcoded value might fail from time to time.



See: https://filiphric.com/waiting-in-cypress-and-how-to-avoid-it

Also: https://docs.cypress.io/api/commands/wait

Reply:
I changed this to a cy.visit(). Let me know if this works for you 👍